### ISSUE TITLE
Outdated description on how to write plugins

### ISSUE THREAD

**Issue by ebukari** [qbittorrent/search-plugins#6]
The instruction on how to write plugins is outdated. No mention is made of `desc_link` and the description could use a few extra details.

**Comment by sledgehammer999** [comment]
IIRC the wiki can be edited by anyone.

**Comment by Shachar2like** [comment]
I've updated the wiki a bit but mostly design wise.
I didn't change the explanation side since I'm still trying to write a plugin myself but the new layout is a lot clearer.

If someone wants to go over and maybe explain in more details what in general needs to be changed for a new plugin & how to test the new plugin, that would be greatly appreciated

**Comment by Shachar2like** [comment]
ok I give.
I've gone over the code, I see lots of def (definition. sort of like setting up specific commands to run) but I don't see anything that runs it.

oh wait I think I got it.
you either run it manually (see the '[testing your plugin](https://github.com/qbittorrent/search-plugins/wiki/How-to-write-a-search-plugin#testing-your-plugin)' section) or Qbittorrent runs it.

I think I figured out more or less everything, anyone available to explain the code or modification in more details?
